app/routes/session.py

```diff
@@ -126,7 +127,7 @@ def login() -> Response:
             "account_service_log_out_url"
         )
 
-    cookie_session["language_code"] = metadata.language_code
+    cookie_session["language_code"] = metadata.language_code or DEFAULT_LOCALE
```

Comment:
This should be `DEFAULT_LANGUAGE_CODE` not `DEFAULT_LOCALE`

Reply:
Fixed 👍

Reply:
oops, my bad on the ticket